azure/README.md

```diff
@@ -1,264 +1,50 @@
-# Cloud API Adaptor on Azure
+# Cloud API Adaptor (CAA) on Azure
 
-This documentation will walk you through setting up Cloud API Adaptor (CAA) on
-Azure Kubernetes Service (AKS). We will build the pod vm image, CAA's
-application image, deploy one worker AKS, deploy CAA on that Kubernetes cluster
-and finally deploy a sample application that will run as a pod backed by CAA
-pod VM.
+This documentation will walk you through setting up CAA (a.k.a. Peer Pods) on Azure Kubernetes Service (AKS). This will walk you through deploying:
+- One worker AKS
+- CAA on that Kubernetes cluster
+- An Nginx pod backed by CAA pod VM
 
 ## Pre-requisites
 
-### Azure Login
+### Azure login
 
-The image build will use your local credentials, so make sure you have
-logged into your account via `az login`. Retrieve your Subscription ID
-and set your preferred region:
+The image build will use your local credentials, so make sure you have logged into your account via `az login`. Retrieve your Subscription ID and set your preferred region:
```

Comment:
This line can be removed. It's already there as part of podvm image build instructions

Reply:
reworded it.